### DISCUSSION TITLE
微调报错

### DISCUSSION THREAD

**Discussion by ahang0712** [zai-org/ChatGLM3#1171] [Bad Case]
![miss1](https://github.com/THUDM/ChatGLM3/assets/77891730/cea2f047-9ec1-402f-8918-f9ad1cf27369)
![miss2](https://github.com/THUDM/ChatGLM3/assets/77891730/14249966-19ec-4a1b-8231-868c013373c1)
训练后进行验证时报错

**Comment by ahang0712** [comment]
AttributeError:ChatGLMModel'object has no attribute "prefix encoder

**Comment by zRzRzRzRzRzRzR** [comment]
不能用peft的ptuning，现在代码中是用全参的办法做ptuning的，更多问题可以在 #253 问
